guides/release/routing/defining-your-routes.md

```diff
@@ -39,19 +39,19 @@ Router.map(function() {
 });
 ```
 
-Inside your templates, you can use [`{{link-to}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/link-to?anchor=link-to) to navigate between
+Inside your templates, you can use [`<LinkTo>`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/link-to?anchor=link-to) to navigate between
 routes, using the name that you provided to the `route` method.
 
 ```handlebars
-{{#link-to "index"}}<img class="logo">{{/link-to}}
+<LinkTo @route="index"><img class="logo"></LinkTo>
 
 <nav>
-  {{#link-to "about"}}About{{/link-to}}
-  {{#link-to "favorites"}}Favorites{{/link-to}}
+  <LinkTo @route="about">About</LinkTo>
+  <LinkTo @route="favorites">Favorites</LinkTo>
 </nav>
 ```
 
-The `{{link-to}}` helper will also add an `active` class to the link that
+The `<LinkTo>` component will also add an `active` class to the link that
```

Comment:
active CSS class or JS class? Should we clarify?

Reply:
It is CSS class (JS class doesn't make sense in this scenario), not sure if we should clarify

Reply:
I should have worded that better- while _I_ am aware of what it meant, I think perhaps we should consider clarifying for a lower cognitive load on the developer reading the docs.